### PR TITLE
auth

### LINKED ISSUE #24: Auth Endpoint Implementation
POST	{{domain}}/user/login	-	Admin login
POST	{{domain}}/user/signup	-	Create a admin account

### PR DESCRIPTION
**Pull request by Bawanthathilan**
## Purpose

The purpose of this PR is to fix #24 

## Goals
Auth Endpoint Implementation

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have performed a self-review of my own code
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/api/auth/auth.controller.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import AuthService from "./auth.service";
+import {HTTP_STATUS} from "../../utils/http";
+import logger from "../../utils/logger";
+import passport from "passport";
+
+const adminRegister = async (req, res) => {
+    logger.info('auth.controller.js createAdmin(): ' + JSON.parse(req.body));
```
Comment (Shehanka) [inline src/api/auth/auth.controller.js:7]: don't log all the things cuz body contains password also 😂

**Review by Shehanka — CHANGES_REQUESTED** [review]

**Comment by Shehanka** [comment]
@sithumonline you do the changes?

**Comment by sithumonline** [comment]
> @sithumonline you do the changes?

yp

**Comment by sithumonline** [comment]
switch to #41

**Pull request closed without merging**